pom.xml

```diff
@@ -551,6 +551,18 @@
             </properties>
         </profile>
         <profile>
+            <id>cloudera</id>
+            <properties>
+                <spark.version>2.0.0.cloudera2</spark.version>
```

Comment:
In CDH, Spark 2 requires Scala 2.11. You're right this could also set Scala 2.11 properties, if that's what you mean. Really, the value of this profile is adding the repository, because with that I can also just set `-Dspark.version=2.0.0.cloudera2`